frontend/src/__tests__/cypress/cypress/pages/pipelines/topology.ts

```diff
@@ -169,8 +177,8 @@ class PipelineDetails extends PipelinesTopology {
     return new DashboardCodeEditor(() => cy.findByTestId('pipeline-dashboard-code-editor'));
   }
 
-  findPageTitle() {
-    return cy.findByTestId('app-page-title');
+  findPageTitle(timeout = 10000) {
+    return cy.findByTestId('app-page-title', { timeout });
   }
```

Comment:
Timeouts should use the default but allow for overriding.
```suggestion
  findPageTitle(timeout?: number) {
    return cy.findByTestId('app-page-title', { timeout });
  }
```